acceptance/allocator_terraform/main.tf

```diff
@@ -13,6 +13,7 @@ resource "google_compute_instance" "cockroach" {
   disk {
     image = "${var.gce_image}"
     size = "${var.cockroach_disk_size}" # GB
+    # type    = "pd-ssd" # uncomment for persistent SSD
```

Comment:
Can this be a flag passed to the test? Seems like the `cockroach_binary` terraform var can be set via the `--at.cockroach-binary` flag.